vulcanai2/models/basenetwork.py

```diff
@@ -612,7 +646,8 @@ def cross_validate(self, data_loader, k, epochs,
             figure_path=figure_path)  # TODO: deal with repeated default parameters
 
     # TODO: Instead of self.cpu(), use is_cuda to know if you can use gpu
-    def forward_pass(self, data_loader, convert_to_class=False):
+    def forward_pass(self, data_loader, convert_to_class=False,
+                     device=None, non_blocking=False):
```

Comment:
I think you may have forgotten to remove the `non_blocking` parameter

Reply:
yeah I forgot to remove that